tests/staggeredrenderer/test_staggeredrenderer.cpp

```diff
@@ -52,12 +52,9 @@ void test_StaggeredRenderer::mapSize()
 {
     StaggeredRenderer renderer(mMap);
 
-    TileLayer *tileLayer = static_cast<TileLayer*>(mMap->layerAt(0));
-    QSize mapSize = renderer.mapSize();
+    QRect mapBoundingRect = renderer.mapBoundingRect();
 
-    QCOMPARE(mapSize, QSize(10 * 64 + 32, 10 * 16 + 16));
-    QCOMPARE(renderer.boundingRect(tileLayer->bounds()),
-             QRect(QPoint(), mapSize));
```

Comment:
Why are you removing the test of `StaggeredRenderer::boundingRect`? It should be possible to just change it to:
```c++
QCOMPARE(renderer.boundingRect(tileLayer->rect()), mapBoundingRect);
```